nipoppy/trackers/run_tracker.py

```diff
@@ -48,6 +51,24 @@ def run(global_configs, dash_schema_file, pipelines, session_id="ALL", run_id="1
     """
     DATASET_ROOT = global_configs["DATASET_ROOT"]
 
+    # load the doughnut (create a new one if needed)
+    fpath_doughnut = f"{DATASET_ROOT}/scratch/raw_dicom/doughnut.csv"
+    if not Path(fpath_doughnut).exists():
+        warnings.warn(f"No doughnut file found, creating one at: {fpath_doughnut}")
+        run_make_doughnut(global_configs, regenerate=True, empty=False)
+    df_doughnut = load_doughnut(fpath_doughnut)
+
+    # load the manifest
+    fpath_manifest = f"{DATASET_ROOT}/tabular/{FNAME_MANIFEST}"
+    df_manifest = load_manifest(fpath_manifest)
+
+    # add BIDS ID column to the manifest
+    df_manifest = df_manifest.merge(
+        df_doughnut,
+        on=[COL_SUBJECT_MANIFEST, COL_SESSION_MANIFEST],
```

Comment:
might be an issue if there is `bids_id` present in the manifest which would create duplicate names with `x` and `y` prefixes in pandas.

Reply:
Good catch! 😅 In that case should we use the `bids_id` column from the manifest or the doughnut?